### DISCUSSION TITLE
How to get 'First Day Of The Week' from Exchange.

### DISCUSSION THREAD

**Discussion by dmitrysync** [ecederstrand/exchangelib#1005] [General]
Hi.
I want to get `First Day Of The Week` from Exchange.
I visited Options -> Calendar -> Personalisation -> Calendar Appearance and saw `Show the first day of the week as`.
Can you help me to get this option from an Exchange server using  `exchangelib`?

**Comment by ecederstrand** [comment]
Is this a setting you found in Outlook? I don't think this is something that's exposed via EWS.

**Reply by dmitrysync** [reply]
Yes. Thanks.
